lib/node_modules/@stdlib/random/strided/laplace/README.md

```diff
@@ -0,0 +1,226 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Beta Random Numbers
+
+> Fill a strided array with pseudorandom numbers drawn from a [laplace][@stdlib/random/base/laplace] distribution.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var laplace = require( '@stdlib/random/strided/laplace' );
+```
+
+#### laplace( N, alpha, sa, beta, sb, out, so\[, options] )
```

Comment:
Copy-paste error: Need to update distribution parameters here and elsewhere in the `README.md` from `alpha` and `beta` to `mu` (mean) and `b` (scale parameter) as in implementation.